features/step_definitions/us152_search_for_deed_via_title_number_and_md_ref.rb

```diff
@@ -0,0 +1,13 @@
+When(/^I search the deed store with title_number "([^"]*)"$/) do |title_number|
+  # Open the URL in browser to check information
+  url_head = 'http://10.10.10.10:9020/deed?md_ref=e-MD12344&title_number='
+  full_url = url_head + title_number
+  visit(full_url)
+end
```

Comment:
I would be tempted to call the API using the method that has been done in other tests (i.e. using the httprb API calls) @response = HTTP.get(full_url). Just to keep consistency of approach in how things are done in acceptance tests. It will also allow you to interrogate the returned data a bit more precisely, rather than relying on checking its part of a page content, as with APIs I would think it's more important that not only is the data present but the format of it is correct. Give us a shout if you want me to talk through this with you.